spec/support/login_macros.rb

```diff
@@ -0,0 +1,6 @@
+module LoginMacros
+  def login_customer(customer = Customer.create!(email: 'jane.doe@codeplay.com.br', password: '123456'))
```

Comment:
O que acham de usar o fabricate aqui? porque aí se esse email for repetido o sequence ajuda a ser mais rápido.

```suggestion
  def login_customer(customer = Fabricate(:customer))
```